src/org/usfirst/frc/team1076/robot/commands/DriveForwardBackward.java

```diff
@@ -0,0 +1,65 @@
+package org.usfirst.frc.team1076.robot.commands;
+
+import org.usfirst.frc.team1076.robot.Robot;
+import org.usfirst.frc.team1076.robot.subsystems.LeftRightMotors;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ * command for driving forwards or backwards
+ * 
+ * @author Ella
+ *
+ */
+public class DriveForwardBackward extends Command {
+
+	LeftRightMotors leftRight;
+	double targetTime;
+	double time;
+	double speed;
+
+	/**
+	 * 
+	 * @param leftRight
+	 *            is the motor object to control.
+	 * @param targetTime
+	 *            measured in seconds.
+	 * @param speed
+	 *            is in the range of -1 to 1.
```

Comment:
It might be worth mentioning that positive is forward, but that's probably obvious.